test/integration-tests/shared/protocolHelpers/seaport.ts

```diff
@@ -57,7 +79,11 @@ export function getOrderParams(apiOrder: any): { order: Order; value: BigNumber
   return { order, value }
 }
 
-export function getAdvancedOrderParams(apiOrder: any): { advancedOrder: AdvancedOrder; value: BigNumber } {
```

Comment:
what do you think about still returning `value` from this function, and having the item type filter be an optional parameter?

Reply:
I would be fine with that, I need to remove `criteriaResolvers` as well as I don't think that's in the api response as well